### ISSUE TITLE
hbal should exit with error code '0' on rebalances cancelled by user

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#442]
Originally reported of Google Code with ID 386.

```
What software version are you running? Please provide the output of "gnt-
cluster --version" and "gnt-cluster version".

2.6

<b>What steps will reproduce the problem?</b>
1. start 'hbal -X'
2. interrupt with Control-C
3. 'echo $?' will return 1

<b>What is the expected output? What do you see instead?</b>

User request is not an error strictly speaking, so hbal should exit with error code '0'.

```
Originally added on 2013-02-22 21:32:10 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
Since this is small, it can still go in 2.7; I'll work on it.
```
Originally added on 2013-03-05 12:07:22 +0000 UTC.

Added Labels: Component-htools Type-Enhancement
Changed State: Accepted
Added to Milestone: Release2.7

**Comment by Ganeti-Issues-Migrator** [comment]
```
FYI, it turns out that the man page already pointed to this model in the 'EXIT STATUS' section, so most likely we wanted to make the change a while ago.
```
Originally added on 2013-03-06 09:55:43 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
commit 2d6bdcc
Author: Iustin Pop <iustin@google.com>
Date:   Wed Mar 6 10:58:53 2013 +0100

    Change hbal behaviour in case of early exit
    
    Currently, hbal exits with status 1 if early exit is requested, even
    when all jobs are successful. This is counter-intuitive behaviour, so
    let's fix it (Issue 386).
    
    Note that the man page had conflicting information already, so it's a
    good thing to clean this up.
    
    Signed-off-by: Iustin Pop <iustin@google.com>
    Reviewed-by: Guido Trotter <ultrotter@google.com>

```
Originally added on 2013-03-08 15:50:11 +0000 UTC.

Mentioned Issue with GoogleCode ID 386 has been migrated to GitHub with ID #442.
Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
Marking 2.7 bugs fixed, as 2.7.0~rc1 was released, and should be ready to be tested in production environments.
```
Originally added on 2013-05-09 05:42:48 +0000 UTC.

Changed State: Released
